erpnext/stock/doctype/item/item.json

```diff
@@ -24,6 +24,7 @@
   "allow_alternative_item",
   "is_stock_item",
   "include_item_in_manufacturing",
+  "grant_commission",
```

Comment:
Can this be moved to the `Sales Details` section ? If required users can Customize the position later.

Reply:
Sure :)

Reply:
Done
![Bildschirmfoto 2021-09-23 um 12 41 32](https://user-images.githubusercontent.com/14891507/134493873-cd99e7f7-1e89-434e-aae8-9da3e2dac94d.png)